closure/webfiles/web_library.bzl

```diff
@@ -209,6 +209,8 @@ web_library = rule(
     executable = True,
     attrs = {
         "path": attr.string(),
+        "host": attr.string(default = "0.0.0.0"),
+        "port": attr.string(default = "6006"),
```

Comment:
I just noticed that this is `attr.string` instead of `attr.int` which requires usages to pass `"8080"` instead of `8080`.